dbt/models/default/default.vw_pin_permit.sql

```diff
@@ -29,7 +29,7 @@ SELECT
     permit.user28 AS local_permit_number,
     permit.permdt AS date_issued,
     permit.certdate AS date_submitted,
-    permit.udate3 AS date_updated,
+    COALESCE(permit.udate3, permit.wen) AS date_updated,
     permit.udate2 AS estimated_date_of_completion,
     permit.user31 AS assessment_year,
```

Comment:
I think we have not actually discussed it yet! `assessment_year` is pretty weird, it seems to act like `year` but it's null 18% of the time. I'm going to ask our subject matter experts what it means, but for now the choice of `assessment_year` is intentional so that we don't accidentally imply to data consumers that this field should be treated the same as `year` in other datasets.